packages/cactus-plugin-keychain-memory/src/test/typescript/unit/plugin-keychain-memory-api-client.test.ts

```diff
@@ -0,0 +1,138 @@
+import test, { Test } from "tape-promise/tape";
```

Comment:
Can this test be written in `jest` instead of `tape`? Also tape tests must be listed in `jest.config` (and probably in `.taprc` as well)